src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+
+  &__image {
+    width: 160px;
+    height: 134px;
```

Comment:
The height of the image should not be fixed for the same reasons mentioned for the width. A fixed height can distort the image when the aspect ratio changes.